src/appFunctions.js

```diff
@@ -132,11 +131,23 @@ export function getStatusNameAdditionOfSettings(name, statuses) {
 }    
 
 export function getStateStatus(name, statuses) {
-    return !statuses.find(status => status.name === name).enabled;
+    const status = statuses.find(status => status.name === name);
```

Comment:
omg
`.find` returns `undefined` or an object based on it you can return `true` or `false`, no need for additional lines

Reply:
should i do so?

export function getStateStatus(name, statuses) {
    if (statuses.find(status => status.name === name).enabled) {
        return true;
    } else {
        return false;
    }    
}

Reply:
@DmitryApan it's not one line solution
google about coercion in js